CHANGELOG.md

```diff
@@ -73,6 +73,21 @@ Difference for SPIR-V passthrough:
 ```
 This allows using precompiled shaders without manually checking which backend's code to pass, for example if you have shaders precompiled for both DXIL and SPIR-V.
 
+#### `EXPERIMENTAL_*` features now require unsafe code to enable
+
+We want to be able to expose potentially experimental features to our users before we have ensured that they are fully sound to use.
+As such, we now require any feature that is prefixed with `EXPERIMENTAL` to have a special unsafe token enabled in the device descriptor
+acknologing that the features may still have bugs in them and to report any they find.
```

Comment:
```suggestion
acknowledging that the features may still have bugs in them and to report any they find.
```

Reply:
Oops, seems I forgot to actually push, kpreid caught this prevoiusly